lib/util.h

```diff
@@ -74,6 +75,7 @@
 #define MSP_ERR_DTWF_ZERO_POPULATION_SIZE                           -38
 #define MSP_ERR_DTWF_UNSUPPORTED_BOTTLENECK                         -39
 #define MSP_ERR_BAD_PROPORTION                                      -40
+#define MSP_ERR_BAD_BINARY_SEARCH_QUERY                             -41
```

Comment:
Excellent - just need to define the string that goes with this in ``util.c`` and we're done.

Reply:
removed this error code since currently the binary search does not return errors (it is guaranteed to return a valid index, as discussed above).